src/Tickets/Emails/Preview_Data.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Tickets Emails Preview Data class.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+/**
+ * Class Preview_Data.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Preview_Data {
+
+	/**
+	 * Get Order preview data.
+	 * 
+	 * @since TBD
+	 *
+	 * @param string $status Status of order.
+	 * 
+	 * @return array
+	 */
+	static function get_order( $status = 'success' ) {
```

Comment:
We must declare visibility. And return type. I'd follow the pattern of receiving an array as parameter, and then looking for the `success` key.

Reply:
I changed all the methods to where it accepts an `$args` parameter so that you can override any of the array items. Good call!